src/templates/ratings/add.html

```diff
@@ -27,7 +27,7 @@
       <p class="rating-label">{{ _('Please select a star rating.') }}</p>
       <textarea id="review-body" rows="2" cols="40" name="body"
                 placeholder="{{ _('Tell us what you love about this app!') }}"
-                required maxlength="150">{{ existingReview.body }}</textarea>
+                required maxlength="150" required>{{ existingReview.body }}</textarea>
```

Comment:
The `required` attribute already appears to be on here. Perhaps we could use the [`pattern` attribute](http://caniuse.com/#search=pattern)? Something like `pattern="^[^\s]+$"` might work well.